### ISSUE TITLE
" Finish Story Mode " button in Stand > Game > Unlocks

### ISSUE THREAD

**Issue by S7NX** [calamity-inc/Stand-Feedback#200]
**Is your feature request related to a problem? Please describe.**
its annoying to load a 100% save file manually, so i wanted to make a suggestion to complete story mode through stand unlocks, and another issue with the save files is that it changes the character's outfit, car modification. so looking for a feature that completes the story mode without changing the character's, cars... etc.

**Describe the solution you'd like**
Basically a simple button called " Finish Story Mode ", in the Game > Unlocks. That would Complete all the quests, and wont change the user's modifications ( character outfit, car mods, etc.)

**Describe alternatives you've considered**
Alternative is to load 100% save file but it changes user modifications so not really preferred.
